docs/index.rst

```diff
@@ -3,12 +3,13 @@ Welcome to Robotic Leader Follower's documentation!
 
 Robotic Leader Follower is a Wayne Sttate University senior capstone project to
```

Comment:
We should probably fix this typo at some point but it doesn't matter too much to me for this PR.